components/tenant-mgt/org.wso2.carbon.tenant.keystore.mgt/src/main/java/org/wso2/carbon/keystore/mgt/KeyStoreGenerator.java

```diff
@@ -213,7 +213,7 @@ private void persistKeyStore(KeyStore keyStore, X509Certificate PKCertificate)
 
             String keyStoreName = generateKSNameFromDomainName();
             // Use the keystore using the keystore admin
-            KeyStoreAdmin keystoreAdmin = new KeyStoreAdmin(tenantId, govRegistry);
```

Comment:
If we are removing the registry reference here, we should be able to remove all the registry imports. And thats part of the main objectives.